src/trans-word-level/instantiate_word_level.cpp

```diff
@@ -31,6 +32,27 @@ std::string timeframe_identifier(
   return id2string(identifier)+"@"+std::to_string(timeframe);
 }
 
+/*******************************************************************\
+
+Function: timeframe_symbol
+
+  Inputs:
+
+ Outputs:
+
+ Purpose:
+
+\*******************************************************************/
+
+symbol_exprt timeframe_symbol(
+  unsigned timeframe,
+  const symbol_exprt &src)
```

Comment:
Take this one by value so that you can move into it.

Reply:
done